numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +2001,205 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_type(self):
+        # Test of inplace additions
+        for t in self.othertypes:
```

Comment:
I think so. You can see the TypeError with, e.g.
```
n [8]: a = ones(3, int8)

In [9]: true_divide(a, 1, a, casting='same_kind')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-9-c3a7900a40e1> in <module>()
----> 1 true_divide(a, 1, a, casting='same_kind')

TypeError: ufunc 'true_divide' output (typecode 'd') could not be coerced to provided output parameter (typecode 'b') according to the casting rule ''same_kind''
```
I think in numpy 1.10 this is always an error.